db/migrate/20150720142931_reregister_archived_whitehall_artefact_routes.rb

```diff
@@ -0,0 +1,12 @@
+class ReregisterArchivedWhitehallArtefactRoutes < Mongoid::Migration
+  def self.up
+    Artefact.where(:state => 'archived', :owning_app => 'whitehall').each do |artefact|
+      # register the routes with the router.
+      RoutableArtefact.new(artefact).register
+    end
+    RoutableArtefact.new(nil).commit
```

Comment:
Not relevant to the review of this PR, but for my own knowledge, why do we call `RoutableArtefact.new(nil).commit` here?